contracts/mock/MockKyberVotingPowerStrategy.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+contract MockKyberVotingPowerStrategy {
+  function validateProposalCreation(uint256 /*startTime*/, uint256 /*endTime*/)
+    external
+    pure
+    returns (bool)
+  {
+    return true;
```

Comment:
Pls add an variable to control `VotingPowerStrategy` return true or false.

Also add a test if `VotingPowerStrategy` return false.